src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -167,6 +187,9 @@ def __runner():
                         self.blue_full_system_proto_unix_io.send_proto(
                             RobotStatus, robot_status
                         )
+                        self.primitive_set_buffer.get(
```

Comment:
The returned value seems to be discarded. Is this buffer needed?

Reply:
Discussed offline, a comment should be added.

Reply:
comment added